lib/srv/exec_test.go

```diff
@@ -282,21 +283,20 @@ func (s *ExecSuite) TestContinue(c *check.C) {
 	cmd, err := ConfigureCommand(ctx)
 	c.Assert(err, check.IsNil)
 
-	// Create a context that will be used to signal that execution is complete.
-	doneContext, doneCancel := context.WithCancel(context.Background())
+	// Create a channel that will be used to signal that execution is complete.
+	cmdDone := make(chan error, 1)
 
 	// Re-execute Teleport and run "ls". Signal over the context when execution
```

Comment:
nit: Is this signalling over a channel rather than a context now?

Reply:
Yes, and also bubbles up the error from `cmd.Run()`.
It's unsafe to call `c.Assert` from goroutines.